web/tasks/XSum.md

```diff
@@ -10,36 +10,43 @@ motivation: TODO.
 
 ## Dataset Description
 
-- **Homepage:**
-- **Repository:**
-- **Paper:**
-- **Point of Contact:**
+- **Homepage:** 
+- **Repository:** https://github.com/EdinburghNLP/XSum
+- **Paper:** [Original Paper](https://arxiv.org/abs/1808.08745)
+- **Point of Contact:** [Shashi Narayan](shashi.narayan@gmail.com)
 
 ### Dataset and Task Summary
 
-[More Information Needed]
+The dataset is for the task of abstractive summarization in its extreme form, its about summarizing a document in a single sentence. It introduces extreme summarization, a new single-document summarization task which does not favor extractive strategies and calls for an abstractive modeling approach. The idea is to create a short, one-sentence news summary answering the question "What is the article about?".
 
 ### Why is this dataset part of GEM?
 
 [More Information Needed]
```

Comment:
This dataset is part of the GEM benchmark for the task of summarization, alongside MLSum and WikiLingua, and acts as a large-scale, high-quality resource for extreme summarization.